lib/compat/wordpress-6.1/block-patterns/centered-footer-with-social-links.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Footer: Centered footer with social links.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'      => _x( 'Centered footer with social links', 'Block pattern title' ),
+	'blockTypes' => array( 'core/template-part/footer' ),
+	'categories' => array( 'footer' ),
+	'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"4em","bottom":"2em"}}},"layout":{"inherit":false}} -->
+					<div class="wp-block-group alignfull" style="padding-top:4em;padding-bottom:2em"><!-- wp:site-title {"textAlign":"center","style":{"typography":{"fontStyle":"normal","fontWeight":"700"}}} /-->
+					
+					<!-- wp:site-tagline {"textAlign":"center"} /-->
+					
+					<!-- wp:spacer {"height":8} -->
```

Comment:
Currently, the spacer block needs a unit in the height attribute to be valid. 
`<!-- wp:spacer {"height":"8px"} -->`
Was this removed on purpose? I suppose it depends what version of Gutenberg/Core that the pattern needs to support.

Reply:
Good eye. I bet that was accidental.

Reply:
Fixed a few of these in 453c77ff553c132efb3bd79a8156de654eeacd49